kong-mesh/kong-mesh-sidecar-scc-cni.yaml

```diff
@@ -0,0 +1,38 @@
+allowHostDirVolumePlugin: false
```

Comment:
this quickstart is specifically for CNI, and used in the Kong docs, so this file cannot be renamed, it breaks the demo

Reply:
I don't see any description of usage of CNI on the quickstart tutorial at https://docs.konghq.com/mesh/latest/features/openshift-quickstart/#main

So I created a new file for the CNI mode to use as I thought the old one was for the "kuma-init" mode.

Reply:
So, is there a planned update for that tutorial? If not, I'll try to update it according to my testing.

Reply:
it is not explicitly called out in this description but it is enabled in the values file https://github.com/Kong/kong-mesh-quickstart-openshift/blob/main/kong-mesh/values.yaml#L254

We can update the doc to clarify that point if it helps

Reply:
Yeah, the doc was well written with a working `values.yaml` provided. 
I didn't use that `values.yaml` because I wanted to customize my own installation, so I wasn't aware the "kuma-cni" should be enabled.